src/content/docs/installation/installation_prepare.mdx

```diff
@@ -11,6 +11,10 @@ import ImageComponent from '~/components/image-component.astro';
 Before preparing the installation, make sure that the computer used meets the minimum system requirements in order to run CachyOS. The installer
 uses an online installation process so a stable and relatively fast internet connection is mandatory.
 
+:::note[Virtual Machines]
+It is not recommended to install CachyOS on a virtual machine. VMs can have issues with incorrect configuration or can be broken entirely. It is recommended to install CachyOS on bare metal.
```

Comment:
Maybe we’ll just limit the support to QEMU/KVM? Because most of the issues happen with VirtualBox and VMware.